cmd/provision.go

```diff
@@ -60,26 +66,35 @@ This ensures that your project ports are properly exposed and externally accessi
 			log.Fatal("remote with name already exists")
 		}
 
-		// Load flags
+		// Load flags for credentials
 		fromEnv, _ := cmd.Flags().GetBool("from-env")
-		instanceType, _ := cmd.Flags().GetString("type")
+		profilePath, _ := cmd.Flags().GetString("profile-path")
+
+		// Load flags for setup configuration
 		user, _ := cmd.Flags().GetString("user")
+		instanceType, _ := cmd.Flags().GetString("type")
```

Comment:
I was gonna complain about these flag getters are really repeative, but I couldn't find a better solution. Do you know any?

Reply:
The oneline is required for sure, and I think it's fine - the err check is meh.

I thought about it again, and I realized we **can** ignore the err check probably - broken inputs will return a 0-value (for strings, it's `""`) which is safe. Doh. Maybe this calls for a reopening of #315? what do you think?